iOS/SnapKit/PhotosExampleWithSnapkit/PhotosExampleWithSnapkit/CustomCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  CustomCell.swift
+//  PhotosExampleWithSnapkit
+//
+//  Created by 이조은 on 2022/09/20.
+//
+
+import UIKit
+
+class CustomCell: UITableViewCell {
+    
+    static let cellId = "customCell"
+    
+    let photoView = UIImageView()
+    let nextBtn = UIButton()
+    
+    //let nextIcon: UIImage = UIImage(named: "chevron.right")!
+    //nextBtn.setImage(nextIcon, for: .normal)
+    
+    //스토리보드로 셀을 작성하게 될 경우 초기화를 해주기 때문에 안해줘도 되지만 코드로 작성하게 될 경우에는 작성해야 한다.
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setLayouts()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    private func setLayouts() {
+        setViewHierarchy()
+        setConstraints()
+    }
+
+    private func setViewHierarchy() {
+        self.addSubview(photoView)
+        self.addSubview(nextBtn)
```

Comment:
UITableViewCell에는 contentView가 기본적으로 올라가며, 여기 contentView에 뷰 요소들을 올려야 유저와 상호작용이 가능합니다.

```suggestion
        self.contentView.addSubview(photoView)
        self.contentView.addSubview(nextBtn)
```